tests/infrared/16/infrared-openstack.sh

```diff
@@ -122,5 +120,8 @@ else
   stf_create_config
   ir_create_overcloud
   ir_expose_ui
+fi
+
+if ${RUN_WORKLOAD}; then
```

Comment:
Ah yes, this ALSO bit me the other day. I think I was missing certain ir plugins or something to make it work

Reply:
I've never see the launch_workload option working properly without any issue. It always stopped sooner or later, sometimes after uploading the cirrus image, sometimes during floating IP creation.